cypress/integration/sidebar_spec.js

```diff
@@ -0,0 +1,17 @@
+describe("Draw", () => {
+  beforeEach("", () => {
+    cy.visit("/graph");
+  });
+  it("Switch to Aboriginal Studies", () => {
+    cy.get("#sidebar-button").click();
+    cy.contains("Aboriginal").click();
+    cy.get("#sidebar-button").click();
+  });
+  it("navigation buttons", () => {
+    cy.get("#zoom-in-button")
+      .click()
+      .click();
+    cy.get("#pan-right-button").click();
```

Comment:
This is a bit funky. If we do `cypress run` (headless mode) this test fails because the button is disabled. Could be because of the width/height dimensions of the SVG vs. the window; not sure how that works in this context. For now you could just disable this check, or you could look into it.